src/App.css

```diff
@@ -81,3 +81,7 @@ a {
 ::selection {
   background-color: #81c784;
 }
+
+.MuiIconButton-root:hover {
```

Comment:
Have you considered using more semantic class names for buttons like this over the "stock" material class names? Maybe `btn--primary`? Also, you may want to store your colors in a variables file.